Code/GraphMol/FileParsers/sgroup_test_data/repeat_groups_query1.mol

```diff
@@ -0,0 +1,29 @@
+
+  Mrv1824 06192020192D          
+
+  0  0  0     0  0            999 V3000
+M  V30 BEGIN CTAB
+M  V30 COUNTS 6 6 1 0 0
+M  V30 BEGIN ATOM
+M  V30 1 C -1.25 0.7484 0 0
+M  V30 2 C -2.5837 -0.0216 0 0
+M  V30 3 C -2.5837 -1.5617 0 0
+M  V30 4 C -1.25 -2.3317 0 0
+M  V30 5 C 0.0837 -1.5617 0 0
+M  V30 6 C 0.0837 -0.0216 0 0
+M  V30 END ATOM
+M  V30 BEGIN BOND
+M  V30 1 1 1 2
+M  V30 2 1 2 3
+M  V30 3 1 3 4
+M  V30 4 1 4 5
+M  V30 5 1 5 6
+M  V30 6 1 1 6
+M  V30 END BOND
+M  V30 BEGIN SGROUP
+M  V30 1 SRU 0 ATOMS=(1 1) XBONDS=(2 1 6) XBHEAD=(2 6 1) XBCORR=(4 6 6 1 1) -
```

Comment:
This is not a real life example, is it?

I wouldn't expect both XBONDS to be listed in XBHEAD, and neither having each bond paired with itself in XBCORR. I'd rather think this would be something like:

```suggestion
M  V30 1 SRU 0 ATOMS=(1 1) XBONDS=(2 1 6) XBHEAD=(1 1) XBCORR=(2 1 6)
```

(same in the Python test, too)